spec/element_spec.rb

```diff
@@ -45,6 +45,23 @@ class PageWithAFewElements < SitePrism::Page
     expect(page).to respond_to :all_there?
   end
 
+  it 'checks only expected_elements if excluded_elements specified' do
+    class PageWithAFewElements < SitePrism::Page
+      element :bob, 'a.b c.d'
+      element :success_msg, 'span.alert-success'
+
+      def excluded_elements
+        %w[success_msg]
+      end
+    end
+    page = PageWithAFewElements.new
+    allow(page).to receive(:has_bob?).and_return(true)
+    allow(page).to receive(:has_success_msg?).and_return(false)
+    expect(page.all_there?).to be_truthy
```

Comment:
Originally wrote this was redundant, but you're making the call here.

Perhaps instead just verbosely call `page.all_there?` I know the whole stylistic area of our Specs needs a lot of work (RE Let blocks and formatting)

Reply:
In my mind, `expect(page.all_there?).to be_truthy` is a fairly key assertion, perhaps even more important than the two below it, because the idea behind `excluded_elements` is to allow for `all_there?` to return true even when elements specified ahead of time are not present on the page. It just happens to also function as a call to `all_there?` that enables the below two assertions about the stubbed methods.

Arguably this could be two specs, under a `describe` block. That's a good area of improvement for the entire rspec suite, along with wider use of `let` blocks and other matters.